include/GooseEYE/GooseEYE.h

```diff
@@ -367,6 +383,21 @@ xt::xarray<double> heightheight(
   const xt::xarray<int>& fmask,
   bool periodic=true);
 
+// Lineal-path function
```

Comment:
All functions should be added to the Python interface (`GooseEYE/python/main.cpp`)